cli/connect/lua/eval_func_body.lua

```diff
@@ -13,9 +29,26 @@ local function table_pack(...)
     return {n = select('#', ...), ...}
 end
 
+local function format_equal(f1, f2)
+    if #f1 ~= #f2 then
+        return false
+    end
+
+    for i, field in ipairs(f1) do
+        if field.name ~= f2[i].name or field.type ~= f2[i].type then
+            return false
+        end
+    end
+    return true
+end
+
 local ret = table_pack(pcall(fun, unpack(args)))
 if not ret[1] then
-    return yaml.encode({box.NULL})
+    local err = unpack(ret, 2, ret.n)
+    if err == nil then
+        err = box.NULL
+    end
+    return yaml.encode({{error = err}})
```

Comment:
Is it the same behavior as before? I guess that just {err} returned in the case.

Reply:
This is the same behaviour as Tarantool `console.eval`. It is made that way to pass the test with `error('blah blah')` console command.

Reply:
Please, add the test with `Lua` output format type.

We also need some tests with tuples + format and Lua/YAML output type.

Reply:
There are already test added for Lua type:
https://github.com/tarantool/tt/blob/8f4d3182f21d2b8ce663c8bd207c3a7e80b969a4/test/integration/connect/test_connect.py#L949-L956
(I will add some tests for yaml).

Tests for errors for table and lua already exist:
https://github.com/tarantool/tt/blob/8f4d3182f21d2b8ce663c8bd207c3a7e80b969a4/test/integration/connect/test_connect.py#L1224-L1231

https://github.com/tarantool/tt/blob/8f4d3182f21d2b8ce663c8bd207c3a7e80b969a4/test/integration/connect/test_connect.py#L908-L914

Reply:
Thank you, great!